source/includes/aws/_subnetworks.md

```diff
@@ -123,6 +123,60 @@ Retrieve a subnetwork in a given [environment](#administration-environments).
 | `availableIPv4Addreses`<br/>_int_                  | The number of available addresses in the subnet's CIDR block.                                                                   |
 | `vpcId`<br/>_string_                               | The ID of the VPC the subnet is associated with.                                                                                |
 
+<!-------------------- CREATE SUBNETWORKS -------------------->
+
+#### Create Subnetworks
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/aws/test-env/subnetworks?operation=create"
+```
+
+> Request body example for named subnets:
+
+```json
+{
+  "vpcId": "vpc-0d5f70bd6b3bf9013",
+  "subnetsToCreate": [
+    {
+      "name": "newSubnet1",
+      "cidrBlock": "10.0.0.0/25",
+      "availabilityZone": "us-east-1c"
+    },
+    {
+      "name": "newSubnet2",
+      "cidrBlock": "10.0.0.128/25",
+      "availabilityZone": "us-east-1c"
+    }
+  ]
+}
+```
+
+> The above commands return a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/subnetworks</code>
+
+Create a VPC in a given [environment](#administration-environments).
+
+| Attributes                      | &nbsp;                                              |
+| ------------------------------- | --------------------------------------------------- |
+| `vpcId`<br/>_string_            | The ID of the VPC.                                  |
+| `subnetsToCreate`<br/>_string_  | List of subnets to be created for a VPC             |
```

Comment:
I am not 100% sold on the notation here, but it has nothing to do with this PR

Reply:
Type should be a list of objects here instead of a string